app/server/server.sbt

```diff
@@ -13,4 +13,8 @@ packageSummary := "A Bitcoin neutrino node and wallet"
 packageDescription := "Runs a Bitcoin neutrino node and wallet, has functionality " +
   "for many different modes and configuration options, see more at https://bitcoin-s.org/docs/applications/server"
 
-enablePlugins(JavaAppPackaging)
```

Comment:
Is this supposed to be removed?

Reply:
It was moved to here: https://github.com/bitcoin-s/bitcoin-s/pull/2673/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R311